concepticondata/concepticon.tsv

```diff
@@ -3927,3 +3927,11 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3941	TAYRA (Eira barbara)	Animals	Omnivorous animal from South and Central America	Person/Thing	
 3942	WAX	Animals	Organic solid compounds of animals or vegetal origin	Person/Thing	
 3943	WRISTBAND	Clothing and grooming	An article used as an adornment around the wrist	Person/Thing	
+3944	LOWER LIP	The body	The lower fleshy protrusion around the opening of the mouth.	Person/Thing	
+3945	ORION'S CONSTELLATION	The physical world	A constellation of stars visible from around the world.	Person/Thing	
+3946	BABY SLING	Clothing and grooming	A piece of cloth used for carrying an infant or small child around a carer's body.	Person/Thing	
```

Comment:
This concept was introduced without being mentioned in your comment. Mappings would include: Snider-2004-1700-547

Reply:
Yes, indeed!